pkg/aws_config_server/assemble_config.go

```diff
@@ -33,6 +33,12 @@ func (a *ClientIDToAWSRoles) getRoles(ctx context.Context, masterRoles []string,
 			return errors.Wrap(err, "Unable to get list of AWS Profiles")
 		}
 		for _, acct := range accountList {
+
+			// HACK until permission issue gets resolved for the account
+			if *acct.Name == "hca-prod" {
```

Comment:
This project is open source, so we shouldn't hardcode statements like this 😆 . If you won't be working on this after today, I'll work on this PR!

Reply:
The advice I usually get: see if you can "catch" this access denied error type and ignore it.

Reply:
I did that just to get it to run, definitely wasn't going to merge the code with that line, hence the draft PR :D